website/source/docs/vault-integration/index.html.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: "docs"
+page_title: "Vault Integration"
+sidebar_current: "docs-vault-integration"
+description: |-
+  Learn how to integrate with HashiCorp Vault and retrieve Vault tokens for
+  tasks.
+---
+
+# Vault Integration
+
+Many workloads require access to tokens, passwords, certificates, API keys, and
+other secrets. To enable secure, auditable and easy access to your secrets,
+Nomad integrates with HashiCorp's [Vault][]. Nomad Servers and Clients
+coordinate with Vault to derive a Vault token that has access to only the Vault
+policies the tasks needs. Nomad Clients make the token avaliable to the task and
+handle the tokens renewal. Further, Nomad's [`template` block][template] can
+retrieve secrets from Vault making it easier than ever to secure your
+infrastructure. 
+
+Note that in order to use Vault with Nomad, you will need to configure and
+install Vault separately from Nomad. Nomad does not run Vault for you.
```

Comment:
Let's make this a little nicer. Like "At this time, you must run and manage Vault separately. I future version of Nomad will include the ability to automatically run and configure Vault for you" or something nice